rbc/heavydb.py

```diff
@@ -442,6 +442,21 @@ def __init__(self,
         # An user-defined device-LLVM IR mapping.
         self.user_defined_llvm_ir = {}
 
+    def __repr__(self):
+        return (f'{type(self).__name__}(user={self.user!r}, password="{"*"*len(self.password)}",'
+                f' host={self.host!r}, port={self.port}, dbname={self.dbname!r})')
+
+    def reconnect(self, user=None, password=None, dbname=None):
+        """Return another RemoteHeavyDB instance with possibly different
+        connection credentials.
+        """
```

Comment:
Perhaps raise an exception if the server version is < (5, 9) here.

Reply:
No need, the reconnect works fine with any server (see test_reconnect).

The test_non_admin_user requires 5.9+ because SQL "DROP USER.." statement did not work with older versions than 5.9.